Not LGTM! 🙈  (bad Fixture name)
```bash
tests/integration/api_image_test.py:89: in test_commit
    assert img['Container'].startswith(id)
E   AssertionError: assert False
E    +  where False = <built-in method startswith of str object at 0x7f24d3aeacf0>('7f52ce4bc498f278121efa4247509994a12532c088f22bc3dfcba6354c22cb74')
E    +    where <built-in method startswith of str object at 0x7f24d3aeacf0> = ''.startswith
_____________________ CommitTest.test_commit_with_changes ______________________
tests/integration/api_image_test.py:107: in test_commit_with_changes
    assert img['Container'].startswith(cid['Id'])
E   AssertionError: assert False
E    +  where False = <built-in method startswith of str object at 0x7f24d3aeacf0>('f3cad02b1ad03d6e3f0bc5527f2097d8f68d14d2a1656cd59f12151476fd0c83')
E    +    where <built-in method startswith of str object at 0x7f24d3aeacf0> = ''.startswith
```

--- a/hack/make/test-docker-py
+++ b/hack/make/test-docker-py
@@ -22,6 +22,10 @@ if [ -n "$TEST_INTEGRATION_USE_SNAPSHOTTER" ]; then
 	# TODO(vvoland): re-enable after https://github.com/containerd/containerd/pull/9554 is merged and vendored.
 	PY_TEST_OPTIONS="$PY_TEST_OPTIONS --deselect=tests/integration/models_images_test.py::ImageCollectionTest::test_save_and_load"
 	PY_TEST_OPTIONS="$PY_TEST_OPTIONS --deselect=tests/integration/models_images_test.py::ImageCollectionTest::test_save_and_load_repo_name"
+
+	# TODO(vvoland): re-enable after https://github.com/docker/docker-py/pull/3203 is merged and in a tagged release.
+	PY_TEST_OPTIONS="$PY_TEST_OPTIONS --deselect=tests/integration/api_image_test.py::CommitTest::test_commit"
+	PY_TEST_OPTIONS="$PY_TEST_OPTIONS --deselect=tests/integration/api_image_test.py::CommitTest::test_commit_with_changes"
 fi
 (
 	bundle .integration-daemon-start

